cmd/ketch/app_deploy.go

```diff
@@ -195,6 +250,24 @@ func getImageConfigFile(ctx context.Context, kubeClient kubernetes.Interface, ar
 	return img.ConfigFile()
 }
 
+func watchAppReconcileEvent(ctx context.Context, kubeClient kubernetes.Interface, app *ketchv1.App) (watch.Interface, error) {
+	reason := controllers.AppReconcileReason{Name: app.Name, DeploymentCount: app.Spec.DeploymentsCount}
+	selector := fields.Set(map[string]string{
+		"involvedObject.apiVersion": v1betaPrefix,
+		"involvedObject.kind":       "App",
+		"involvedObject.name":       app.Name,
+		"reason":                    reason.String(),
+	}).AsSelector()
+	watcher, err := kubeClient.CoreV1().
+		Events(app.Namespace).Watch(ctx, metav1.ListOptions{FieldSelector: selector.String()})
+	if err != nil {
+		// Shouldn't happen
+		return nil, err
+	}
+	defer watcher.Stop()
```

Comment:
this line stops the watcher immediately

Reply:
thanks, didn't seen it after last modification